components/__tests__/__snapshots__/locale-chooser.js.snap

```diff
@@ -274,21 +265,25 @@ exports[`opened state 1`] = `
           >
             <glamorous(li)>
               <li
-                className="css-l0f4u7"
+                className="css-1uz022"
               >
                 <glamorous(a)
-                  href="about://blank"
+                  aria-label="Change localization to English"
+                  href="about://undefinedblank"
                 >
                   <a
```

Comment:
Also add `role="presentation"` for better interoperability as stated [here](https://www.w3.org/WAI/tutorials/images/decorative/).